cloudvolume/lib.py

```diff
@@ -817,7 +812,7 @@ def normalize_float(img):
     elif index == 2:
       img = image[:, :, level, channel ]
     else:
-      raise NotImplemented
+      raise NotImplementedError
```

Comment:
@william-silversmith please double check whether this is appropriate:
https://docs.python.org/3/library/constants.html#NotImplemented

Reply:
Thanks for that link. I didn't know that NotImplemented was defined as a constant. According to https://docs.python.org/3/library/exceptions.html

```This exception is derived from RuntimeError. In user defined base classes, abstract methods should raise this exception when they require derived classes to override the method, or while the class is being developed to indicate that the real implementation still needs to be added.```

This would imply that your correction from NotImplemented to NotImplementedError is more correct than what I wrote, but that we should probably use some other kind of error like IndexError or a custom error.